ctapipe/containers.py

```diff
@@ -206,6 +207,12 @@ class PeakTimeStatisticsContainer(StatisticsContainer):
     container_prefix = "peak_time"
 
 
+class CoreParametersContainer(Container):
+    """Telescope-wise shower's direction in the TiltedFrame"""
+    container_prefix = "core"
+    psi = Field(nan * u.deg, "Image direction in the Titled/Ground Frame", unit="deg")
```

Comment:
What is it? TiltedGroundFrame or GroundFrame? And I thought this is `TelescopeFrame` / `NominalFrame` not groundframe

Reply:
It is TiltedFrame, but since it comes from the array-CameraFrame when projected it corresponds also to the one on the ground (aka the one actually visualized with ArrayDisplay).

The value in TelescopeFrame (or CameraFrame, depending on the camera geometry in which the image is cleaned and parametrized) in stored in HillasParametersContainer with the rest of the Hillas parameters.

Reply:
I should update this docstring, though.